src/WebJobs.Script/OutOfProc/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -111,37 +119,69 @@ internal void AddProvidersFromAppSettings()
             }
         }
 
+        internal void AddHttpInvokerProviderFromAppSettings()
+        {
+            var httpInvokerSection = _config.GetSection($"{LanguageWorkerConstants.HttpInvokerSectionName}");
+            var workerDirectorySection = httpInvokerSection.GetSection(LanguageWorkerConstants.WorkerDirectorySectionName);
```

Comment:
Is this temporary or is the HTTP invoker path going to be configured by another worker.config.json (or worker.config.json-like file)?

Reply:
this will be configured on host.json tracking issue https://github.com/Azure/azure-functions-host/issues/4929 . Working on updating this PR to include options setup for http invoker. Apologies for the confusion,